src/redux/services/api.ts

```diff
@@ -104,13 +100,7 @@ export const api = createApi({
     }),
 
     // POST: Set project installments
-    setProjectInstallments: builder.mutation<
-      any,
-      {
-        formData: Partial<ProjectInstallment>[];
-        uuid: string;
-      }
-    >({
+    setProjectInstallments: builder.mutation<any, { formData: Partial<ProjectInstallment>[]; uuid: string }>({
```

Comment:
Same as the previous one, did something change here? I can't see it :D

Reply:
Same with this one